### PR TITLE
Added DEV Console page

### PR DESCRIPTION
**Pull request by FroMage**
Docs for https://github.com/quarkusio/quarkus/pull/13899

### CODE REVIEW

File: _guides/dev-console.adoc [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+}
+----
+<1> While you can use https://vertx.io/docs/vertx-web/java/#_routing_by_http_method[any Vert.x handler], 
+    the `DevConsolePostHandler` superclass will handle your POST actions
+    nicely, and auto-redirect to the `GET` URI right after your `POST` for optimal behaviour.
+<2> You can get the Vert.x `RoutingContext` as well as the `form` contents
+<3> Don't forget to add a message for the user to let them know everything went fine
+<4> You can also add error messages
+
+NOTE: the flash messages are handled by the `main` DEV template and will result in nice notifications for your
```
Comment (gastaldi) [inline _guides/dev-console.adoc:333]:
```suggestion
NOTE: Flash messages are handled by the `main` DEV template and will result in nice notifications for your
```

Comment (FroMage) [reply _guides/dev-console.adoc]: Thanks, done.

**Comment by gastaldi** [comment]
This is great, but I think this guide should live in the quarkusio/quarkus repository?

**Comment by FroMage** [comment]
It is there, but it needs to be copied here to visualise it for review, and ultimately to land online.

**Pull request closed without merging**